apps/party-time-backend/src/main/java/com/partytime/api/controller/AuthController.java

```diff
@@ -1,9 +1,78 @@
 package com.partytime.api.controller;
 
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import com.partytime.api.dto.AccountDTO;
+import com.partytime.api.dto.AccountRegisterDTO;
+import com.partytime.jpa.mapper.AccountMapper;
+import com.partytime.service.AuthService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.security.SecurityRequirements;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
+import lombok.RequiredArgsConstructor;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
 
 @RestController
-@RequestMapping("/api/v1/auth")
+@RequestMapping("/api/auth")
+@Validated
+@RequiredArgsConstructor
+@Tag(
+    name = AuthController.TAG,
+    description = "API Endpoints providing all required logic for Authentication"
+)
 public class AuthController {
+    static final String TAG = "Authentication API";
+
+    private final AuthService authService;
+
+    /**
+     * F010 - Konto Erstellen
+     */
+    @PostMapping("/register")
+    @Operation(
+        description = "Register a new Account",
+        responses = {
+            @ApiResponse(
+                description = "The Account Object of the newly created Account",
+                responseCode = "200",
+                useReturnTypeSchema = true
+            ),
+            @ApiResponse(
+                description = "Account with Email already exists",
```

Comment:
@dangllucas Warum ist das hier auf english?

Reply:
Das wäre reine Dokumentation auf der Swagger Seite.
Bedeutet, das würde niemals zum User raus gehen.
Bedeutet, entspricht unseren anforderungen.
Code auf English, Ausgaben in Deutsch